### ISSUE TITLE
Some unwanted symbols on map

### ISSUE THREAD

**Issue by basiljose1** [allenhwkim/angularjs-google-maps#813]
i have attached the image of the issue, i don't know how to reproduce it since it happen rarely

**Comment by basiljose1** [comment]
Any solution to avoid this

**Comment by allenhwkim** [comment]
I don't think this happens with this module, also I  cannot reproduce this on my end.   Please include a plunker example so that I see this issue.

**Comment by basiljose1** [comment]
zoom control button png image is showing , instead of single plus it shows the full image

you can see this in latest chrome browser 
https://www.bagyourjob.com
